config/config.go

```diff
@@ -683,6 +683,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.33across.endpoint", "http://ssc.33across.com/api/v1/hb")
 	v.SetDefault("adapters.33across.partner_id", "")
 	v.SetDefault("adapters.adform.endpoint", "http://adx.adform.net/adx")
+	v.SetDefault("adapters.adhese.endpoint", "https://ads-{{.Host}}.adhese.com/json")
```

Comment:
so the template param should rather be "account" rather than "Host"

Reply:
We seem to be limited to the values of
`
// EndpointTemplateParams specifies params for an endpoint template
type EndpointTemplateParams struct {
	Host        string
	PublisherID string
	ZoneID      string
	SourceId    string
}
`

Reply:
aha, I didn't know that. So "PublisherID" sounds closer to me. Or maybe we should stick to the "Host" but just pass the whole ads-xxx.adhese.com?